### DISCUSSION TITLE
Multiple Forms or nested Form

### DISCUSSION THREAD

**Discussion by kochch1337** [neuefische/questions#353] [Web] [answered]
Hi,

in my current form I want to select some dropdowns and I have some text fields. Additonally I need the function to add items from a dropdownlist to an array and display the array. The problem is, I need the items from the dropdown only to be added after clicking a button. I can't use the submit button, because I need this one for the final form. 

Here is a small image how the form should look
<img width="288" alt="Screenshot 2022-11-30 at 16 54 58" src="https://user-images.githubusercontent.com/113910347/204845993-f45b8a4c-9e42-4f31-b3da-db205d35979a.png">

The developers and the BPs should be selected via dropdown, and after clicking on the + button they should be added to the list/array. Once both lists are filled the form submit (save button) then will perform the submit/save.

I'm not allowed to nest form in form. If i want to use form after form I can't use the needed styling/placing of the components.

**Comment by JessicaLoers** [comment, marked as answer]
**Hi @kochch1337**  👋
To be honest, I'm not sure if I understand your wish/problem correctly. 
Nevertheless I've prepared an [example ](https://codesandbox.io/s/form-with-select-1r8bhx?file=/src/App.js) for you.
